package.json

```diff
@@ -73,15 +73,15 @@
     "@testing-library/react-hooks": "5.1.1",
     "@testing-library/user-event": "12.8.3",
     "@woocommerce/e2e-environment": "0.1.5",
-    "@wordpress/babel-plugin-makepot": "4.1.0",
-    "@wordpress/babel-preset-default": "5.1.0",
-    "@wordpress/e2e-test-utils": "5.1.0",
-    "@wordpress/env": "4.6.0",
-    "@wordpress/eslint-plugin": "9.0.6",
-    "@wordpress/jest-preset-default": "7.0.1",
-    "@wordpress/postcss-themes": "3.0.1",
-    "@wordpress/prettier-config": "1.0.1",
-    "@wordpress/scripts": "16.1.4",
+    "@wordpress/babel-plugin-makepot": "wp-5.9",
+    "@wordpress/babel-preset-default": "wp-5.9",
+    "@wordpress/e2e-test-utils": "wp-5.9",
+    "@wordpress/env": "wp-5.9",
+    "@wordpress/eslint-plugin": "wp-5.9",
+    "@wordpress/jest-preset-default": "wp-5.9",
+    "@wordpress/postcss-themes": "wp-5.9",
+    "@wordpress/prettier-config": "wp-5.9",
+    "@wordpress/scripts": "wp-6.0",
```

Comment:
I keep it as wp-6.0 because the 5.9 is causing errors, so I don't think that makes sense fix the 5.9 considering we need to update soon to 6.0.